src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1597,7 +1597,7 @@ public static function get_sizes() {
 		// The max_execution_time defaults to 0 when PHP runs from cli.
 		// We still want to limit it below.
 		if ( empty( $max_execution_time ) ) {
-			$max_execution_time = 30;
+			$max_execution_time = 30; // 30 seconds.
 		}
 
 		if ( $max_execution_time > 20 ) {
```

Comment:
I didn't touch this line because the comment below explains the units.

However, this condition tests to see if the `max_execution_time` is greater than 20, but the comment states:
> If the max_execution time is set to _lower_ than 20 seconds.

The comment should be changed to reflect what the condition actually checks:

```php
/*
 * If the max_execution_time is set to more than 20 seconds, reduce it a bit to prevent
 * edge-case timeouts that may happen after the size loop has finished running.
 */
```

Also note that previously, the comment was not in the standard multiline comment format per [Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments).